src/style.css

```diff
@@ -1 +1,86 @@
-/* Styles go here */
+
+html {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+header {
+  display: block;
+  padding: 0 50px;
+}
+
+@media screen and (max-width: 1024px) {
+  .container {
+    max-width: 1024px;
+  }
+}
+
+.nav {
+  display: flex;
+}
+
+a {
+  position: relative;
+  text-decoration: none;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  color: #000;
+}
+
+.nav_list {
+  list-style-type: none;
+  display: flex;
+  align-items: center;
+  position: relative;
+  text-transform: uppercase;
+  padding: 0;
+  margin: 0;
+}
+
+body {
```

Comment:
hint: such tags as `html` and `body` better pace in top of the file to see them first because they more global then other